vocab/credentials/v2/mk_vocab.rb

```diff
@@ -287,13 +296,13 @@ def to_ttl
       output << %(  rdfs:isDefinedBy cred: .)
     end
 
-    output << "\n# Instance definitions" unless @instances.empty?
-    @instances.each do |id, entry|
-      output << "cred:#{id} a #{namespaced(entry[:type])};"
-      output << %(  rdfs:label "#{entry[:label]}"@en;)
-      output << %(  rdfs:comment """#{entry[:comment]}"""@en;)
-      output << %(  rdfs:isDefinedBy cred: .)
-    end
+    # output << "\n# Instance definitions" unless @instances.empty?
```

Comment:
why comment these out but not remove?

Reply:
My own caution... The background is that the ruby script is not of my making, I inherited it. Furthermore, I am not really familiar with ruby, so I have to be extremely careful about any modification I do on the code because I am driving in the dark. I.e., I put things in comments instead of boldly deleting it. It would be better if I re-wrote the whole script in a language that I am more familiar with, but I decided it is not worth the trouble.

I expect the v2 vocabulary to go through some more iteration in the coming period; once we get to a final point, it is my plan to clean the script of such impurities...

Reply:
fair enough!